src/plugins/content/accessibility/jumplinks/plugin.js

```diff
@@ -0,0 +1,137 @@
+    init: function init() {
+        const self = this;
+
+        self.jumplinks = jumplinksFactory({})
+            .on('render', function() {
+                handleJumpLinks();
+            });
+        self.shortcuts = shortcutsFactory({});
+
+        function handleJumpLinks() {
+            const _jumpLinksBehavior = {
+                jumpLinkQuestion: {
+                    selector: '.top-action-bar .jump-links-box [data-jump=question] ',
+                    eventName: 'question',
+                    handler: () => {
+                        const e = findFocusable(self.getAreaBroker().getContentArea() );
+                        e && e.focus();
+                    }
+                },
+                jumpLinkNavigation: {
+                    selector: '.top-action-bar .jump-links-box [data-jump=navigation]',
+                    eventName: 'navigation',
+                    handler: () => {
+                        const e = findFocusable(self.getAreaBroker().getNavigationArea() );
+                        e && e.focus();
+                    }
+                },
+                jumpLinkToolbox: {
+                    selector: '.top-action-bar .jump-links-box [data-jump=toolbox]',
+                    eventName: 'toolbox',
+                    handler: () => {
+                        const e = findFocusable(self.getAreaBroker().getToolboxArea() );
+                        e && e.focus();
+                    }
+                },
+                jumpLinkTeststatus: {
+                    selector: '.top-action-bar .jump-links-box [data-jump=teststatus]',
+                    eventName: 'teststatus',
+                    handler: () => {
+                        const e = findFocusable(self.getAreaBroker().getPanelArea() );
+                        e && e.focus();
+                    }
+                },
+                jumpLinkShortcuts: {
+                    selector: '.top-action-bar [data-jump=shortcuts]',
+                    eventName: 'shortcuts',
+                    handler: () => {
+                        self.shortcuts.show();
+                        const closeHandler = function(event) {
+                            if (event) {
+                                self.shortcuts.hide();
+                                self.shortcuts.getElement().off("click", closeHandler);
+                                $(window).off("keydown", closeHandler);
+                            }
+                        };
+                        self.shortcuts.getElement()
+                            .off("click", closeHandler)
+                            .on("click", closeHandler);
+                        $(window)
+                            .off("keydown", closeHandler)
+                            .on("keydown", closeHandler);
+                    }
+                },
+            };
+            _.forOwn(_jumpLinksBehavior, (linkDescription) => {
+                self.jumplinks.on(linkDescription.eventName, () => {
```

Comment:
you do not need so many events. two will be enough:
- one for jump links
- one for shortcuts

please pass additional arguments to the shortcuts event and use area broker based on it
```
const mapJumpToAreaBroker = {
teststatus: 'getPanelArea',
...
}

on('...', (jump) => {
const $element = self.getAreaBroker()[mapJumpToAreaBroker[jump]]();
...
}
```

Reply:
👌